### DISCUSSION TITLE
Custom Toggle Color

### DISCUSSION THREAD

**Discussion by shahaab-ali** [saadeghi/daisyui#1235] [Q&A] [answered]
Hi everyone,

How can we add custom color to toggle component? like success, warning etc

**Comment by saadeghi** [comment, marked as answer]
Similar discussions:  
https://github.com/saadeghi/daisyui/discussions/729  
https://github.com/saadeghi/daisyui/issues/1273
```html
<input
  type="checkbox"
  class="toggle checked:bg-success checked:border-success"
>
```

**Reply by shahaab-ali** [reply]
Thanks
